sdk/python/kfp/dsl/_container_op.py

```diff
@@ -154,5 +155,38 @@ def set_cpu_limit(self, cpu):
     self._validate_cpu_string(cpu)
     self.cpu_limit = cpu
 
+  def set_volumes(self, volumes):
+    """Specifying what K8s volumes the container depends on
+
+    Args:
+      volumes: a list of Kubernetes volumes
+      For detailed spec, check
+      https://github.com/kubernetes-client/python/tree/master/kubernetes/client/models
+    """
+
+    self.volumes = volumes
+
+  def set_volume_mounts(self, volume_mounts):
+    """Specifying how volumes are mounted to the container
+
+    Args:
+      volume_mounts: a list of Kubernetes volume mounts
+      For detailed spec, check
```

Comment:
It's better. It would be clearer if you also specify what type of Python object is expected. For example, does it only accept "k8s_client.V1VolumeMount"? If so, make it explicit. If you were user of this API and knows little about K8s volume mounts, you probably need more help.

Reply:
cool i add the link to the definition file.

Reply:
This would require loading the Kubernetes module for every usage of the API. Do you think that a string annotation might be sufficient?
E.g.

    def set_volume_mounts(self, volume_mounts: 'List[kubernetes.client.models.V1VolumeMount]'):